src/app/scripts/cac/map/cac-map-templates.js

```diff
@@ -333,8 +333,14 @@ CAC.Map.Templates = (function (Handlebars, moment, Utils) {
             return new Handlebars.SafeString(dt.format('h:mma'));
         });
 
-        Handlebars.registerHelper('inMiles', function(meters) {
-            return new Handlebars.SafeString(Math.round(((meters / 1000) * 0.621371) * 100) / 100);
+        Handlebars.registerHelper('inMilesYards', function(meters) {
+            if (meters < 797) {
+                var yards = Math.round(meters * 1.09361);
+                return new Handlebars.SafeString(yards.toString() + ' yards');
```

Comment:
Was there a discussion of what small unit to use and where to set the threshold for using it?  I remember a conversation in RRP which I thought was about feet and the threshold floating around was lower (.2 mi? or maybe it was just that we observed that Google uses .1), but I don't remember it reaching a definitive conclusion.  But for me, the smaller unit in which I think is feet, and I switch to fractional miles lower, probably around a quarter mile.

So I would vote for
- renaming this helper to `formatDistance`
- using feet, with a lower switch-over threshold
- setting this up using a config object, e.g.
```
smallDistanceUnit = {
    thresholdMiles: 0.25,
    unitName: 'feet',
    conversionFromMeters: 3.28084
};
```
(The threshold could be in meters, but it seems nicer to have the code convert it, i.e. `if (meters < smallDistanceUnit.thresholdMiles * 1609.34) {`.)
- put that somewhere where it can be shared between `cac-map-templates.js` and `cac-routing-itinerary.js`. I'm not sure where that would be.  Two identical copies of the config object would still be better than two similar-but-not-identical functions, but one copy would be best.